src/main/java/duke/ToDo.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+/**
+ * The ToDo class contains the name and completion status of the task.
+ */
+public class ToDo extends Task {
+
+    /**
+     * Constructor of a ToDo object.
+     * @param description The description of the task.
```

Comment:
Perhaps you should leave a line between your description and your @param line?

Reply:
Alright thanks